### ISSUE TITLE
`swarm update` should give some idea of what is going on if it fails

### ISSUE THREAD

**Issue by oponder** [giantswarm/cli#439]
I just ran `swarm update` which failed with an error. (error-id: jOqAc/WlIO0)

```
Updating component 'webclient/nginx' with version 'latest' and strategy 'one-by-one'…
- Unknown server error: error-id: jOqAc/WlIO0

Please try again in a moment. If the problem remains, please contact us at support@giantswarm.io.
make: *** [swarm-update] Error 1
```

The first instance updated without a problem. The second instance went to failed state while updating, and then the CLI threw that error.

However, the update continued in the background and the failed instance resolved itself. 

I feel like the text when an error occurs could be more helpful of what might have gone wrong, and if the update is still going on, or if it is completely halted.

**Comment by zyndiecate** [comment]
@ewoutp IIRC you already worked on providing more information to the user when this unexpected errors happen. Do you have a link for us?

**Comment by ewoutp** [comment]
@zyndiecate Not sure what you're referring to.
This is what I can think of:
- This morning I deployed a fix in the app-service&api that puts the actual error message in the `status_message` field of a task when one of its dependencies has failed.
- Long time ago I proposed more details in api-schema to pass errors.
Not sure if either of these apply here.
One thing is sure, users should not see "Unknown server errors" for "normal" and expected errors.
